cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -55,6 +55,9 @@ def _install(self):
         for source in sources.values():
             yum_install(source)
 
+    def _installing_manager(self):
```

Comment:
I think a better name would be "_install_manager"

Reply:
Hmm... can do that- sounds like it's possibly an imperative (telling it to install the manager), but given the context it might be clear enough... thoughts?

Reply:
Changed.